bin/test_sequencer

```diff
@@ -135,4 +135,10 @@ diff -u out/generated.md docs/specs/sequences/generated.md || (
 echo Checking for duplication in out/sequencer.out and etc/sequencer_planning.txt
 bin/test_sequencer_plan
 
+echo Checking that the sequences in validator/sequences are correct
```

Comment:
Wouldn't this be checking the sequences in the generated site_model (out/) are correct (verified against validator/sequences)?

I think part of the problem here is the overall generation/updating of these things is a bit... wonky?  I'm not happy with whatever hack I have in place so some of this stuff is confusing me (even before you touched it at all!)